app/src/androidTest/java/com/github/h3lp3rs/h3lp/MainPageTestActivity.kt

```diff
@@ -134,4 +143,21 @@ class MainPageTestActivity {
             clickingOnButtonWorksAndSendsIntent(FirstAidActivity::class.java, withId(R.id.button_first_aid), true)
         }
     }
+
+    @Test
+    fun clickingOnProPortalButtonGoesToProPortalIfVerifiedUser() {
+        val proUser = ProUser(USER_TEST_ID, "","","")
+        proUsersDb.setObject(USER_TEST_ID,ProUser::class.java,proUser)
```

Comment:
```suggestion
        val proUser = ProUser(USER_TEST_ID, "", "", "")
        proUsersDb.setObject(USER_TEST_ID,ProUser::class.java, proUser)
```

just for consistency